app/src/main/java/org/ieeeguc/ieeeguc/models/Meeting.java

```diff
@@ -50,8 +49,64 @@ public Meeting(int id, Date start_date, Date end_Date, JSONObject goals,
         this.supervisor = supervisor;
         this.created_at = created_at;
         this.updated_at = updated_at;
+    }
 
-
+    /**
+     * This method is called when a user requests to see a meeting details.
+     * @param {int}          id            [Meeting id].
+     * @param {HTTPResponse} httpResponse  [Interface instance].
+     * @param {String}       token         [Authorization token].
+     */
+    public static void getMeeting(int id, final HTTPResponse httpResponse, String token){
+
+        String stringId = String.valueOf(id);
+        OkHttpClient client = new OkHttpClient();
+        Request request = new Request.Builder()
+                .header("Authorization",token)
+                .header("user_agent","Android")
+                .url("http://ieeeguc.org/api/meeting/" + stringId)
```

Comment:
just a quick hint you don't have to change it but for your knowledge only :D.
* you can append an `int` to a `String` directly in Java.